packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -155,20 +153,22 @@ class LayoutProperties {
 
   static BoxConstraints deserializeConstraints(Map<String, Object> json) {
     // TODO(albertusangga): Support SliverConstraint
-    if (json == null || json['type'] != boxConstraintsType.toString())
-      return null;
+    if (json == null || json['type'] != 'BoxConstraints') return null;
     // TODO(albertusangga): Simplify this json (i.e: when maxWidth is null it means it is unbounded)
     return BoxConstraints(
```

Comment:
is there no longer a risk of it being null?

Reply:
did you mean the `maxWidth` and `maxHeight`? Yes, the field will always be there and will be a string with value 'Infinity' if it is unbounded.